### ISSUE TITLE
No status during finetuning

### ISSUE THREAD

**Issue by mgh1** [BVLC/caffe#895]
I am finetuning the caffe reference imagenet model and not seeing any iteration status printed to the terminal window. It's been running for about an hour and half. I am running it in a tmux window and able to detach while running and observe via top and nvidia-smi that indeed the fine-tuning process is running and so is the GPU (99% utilization).

Should I be seeing some kind of output while running in the terminal?

I also couldn't find any solverstate files or log files. Where might I find these?
